### DISCUSSION TITLE
Various points

### DISCUSSION THREAD

**Discussion by DB99** [noDRM/DeDRM_tools#4] [General]
I have been following the discussion on Mobileread and thought I would get the discussion off the ground here.  Fist of all my thanks to noDRM.   And to Apprentice Harper, jhowell, ellekayem and to all involved with the tools project over many years.

The ethical problems have been discussed frequently on Mobileread and thare are many differing views, as there should be.  I agree with the points noDRM made in the first reply on the issue raised by jhowell.  In my view prevention of copying takes second place to lock-in so far as most Vendor's are concerned.  The disadvantages to readers are obvious.

So far as so-called social DRM is concerned I too have no problems with its removal.  Hopefully users will not be lulled into a false sense of security and believe that all identifying information has necessarily been removed.

If philosophical conflicts permit it would be nice to see one project rather than two, though I can see a day coming when development of this type of tool is either abandoned or forced to  the dark web.

**Comment by pdurrant** [comment]
Good luck to noDRM. I hope to continue using DeDRM for many years. At least until book publishers finally do the sensible thing and stop using DRM.

**Comment by j-howell** [comment]
See #2 **Questions for noDRM** for some prior discussion that I mistakenly created as an issue.

**Comment by dougmassay** [comment]
> Giving a purchased book to a close friend or a family member to read is something that's perfectly legal (with paper books and DRM-free eBooks) 

Assumes facts not in evidence. Ebooks purchased DRM-Free are still covered by license and ToS. There is no inherent right (legal, or otherwise) conferred upon the purchaser to lend ebooks that are purchased sans drm.

I happen to agree that people _should_ be able to responsibly lend an ebook like they can a paper book. I'm just under no delusions that I'm legally entitled to do so. Nor do I think it's very responsible to encourage people to believe they're legally entitled to lend ebooks just because they happened to be purchased without drm.

And yes. I disagree heartily with giving users the ability to strip any and all potential personally identifying metadata and/or watermarks. noDrm's arguments for allowing users to do so are not very convincing, IMO.

**Reply by noDRM** [reply]
First of all, just to make sure - I am not a lawyer and this is not legal advice.

I believe that when an online shop sells eBooks, and you put your eBook in a shopping cart, then click buttons like "Buy eBook", then you bought an eBook. Not "a personal, non-transferable license to read a text file that may be withdrawn at any time". Just like with a book. Anything else is a surprising clause which is not enforceable in various jurisdictions (like with the [Consumer Protection Act 68](https://www.kemplaw.co.za/363-unusual-and-or-surpising-clauses-in-a-contract) in South Africa, or the [Council Directive 93/13/EEC of 5 April 1993 on unfair terms in consumer contracts](https://europa.eu/youreurope/citizens/consumers/unfair-treatment/unfair-contract-terms/index_en.htm) in the European Union). 
Maybe contracts like these are enforceable in the US - I have no idea, I'm not an expert in US law - but the US isn't the only country in the world, and there's multiple countries where buying something means you actually buy something, and don't allow re-defining the term "buy" in the Terms and Conditions. And this means you own that particular copy of the eBook. You can't upload it to the internet for everyone to download (that'd be like producing hundreds or thousands of copies of a paper book), but you can give it to family members or friends to read when you're done reading it, just like it's always been done with paper books.

And it looks like at least for software, there are court rulings like [UsedSoft GmbH v. Oracle International Corp., C-128/11, European Court of Justice](https://curia.europa.eu/juris/documents.jsf?num=C-128/11&language=en) that state just that - for example, you being able to re-sell the copy of MS Office that you no longer need, even though Microsoft claims you only bought a personal, non-transferable license. The court decided that that doesn't matter, because in the end user's perspective, they bought "MS Office". Not "a personal, non-transferable license to use MS Office". And according to the court, this also explicitly applies to download licenses where there's no physical disc to sell. So even if you consider buying an eBook being just "buying a license to read that eBook", this includes (in my opinion, I'm not a lawyer) the right to (temporarily or permanently) transfer this license to someone else, at least when this doesn't happen commercially. Because customers can reasonably expect that they are able to lend the book or eBook that they bought to their family members or friends.

Yes, there's no way to verify if you actually deleted the eBook from your hard drive (and to be honest, probably nobody would do that), but that's the same with digital software and the court still decided that selling used software is legal.

I am not encouraging people to believe they're legally allowed to do anything. It's everyone's responsibility to check if what they're doing is legal in their own country. Just like with DRM removal itself, which is legal in some countries and illegal in others. And the watermark removal isn't even enabled by default, so it's even an additional, explicit configuration by the user to deliberately enable this if they want to.

---

And even when completely ignoring the part about sharing eBooks with friends and the whole legal blab attached to that - I would not want to get into legal trouble because a watermarked eBook I bought does make its way into the internet due to no fault of my own. There's lots of ways this can happen. I already mentioned some ways in #2. Just Google around for so-called "open directories", or take a look at [Shodan](https://www.shodan.io/). You are going to find hundreds, if not thousands of results of people who accidentally have their home computer configured to share their Calibre installation with the public. Yes, doing that is technically illegal redistribution, but a very very very large amount of these libraries is public by accident, not in order to share eBooks publicly. If I had that happen to me, I don't need lawyers suing me for "redistribution of ebooks" when I didn't do that deliberately, just because they found my watermark in one of the books in my Calibre library that's open to the public by accident.

---

Watermarks are DRM, even if they don't appear to restrict the user at first glance. I am sure that there are people who disagree with that, but this is my opinion and if people don't like it they can just decide to leave that functionality disabled. Or continue using Apprentice Harper's original version.

**Comment by dougmassay** [comment]
>  I would not want to get into legal trouble because a watermarked eBook I bought does make its way into the internet due to no fault of my own. There's lots of ways this can happen. 

And all of those same "lots of ways this can happen" would prevent you from being prosecuted should it happen due to no fault of your own.

> I am not encouraging people to believe they're legally allowed to do anything.

Then perhaps you should not publicly present your opinion/belief that it is perfectly legal to share drm-free ebooks with family members or close friends as if it were fact (as quoted above, and with no stipulations of jurisdictional variances at the time).

But we're not going to convince each other. So feel free to carry on as you see fit with no further interference from me.

**Reply by noDRM** [reply]
> And all of those same "lots of ways this can happen" would prevent you from being prosecuted should it happen due to no fault of your own.

Probably, but you'd still have a ton of legal issues to deal with. And you'd need to prove that. If you're being sued for sharing eBooks because tons of eBooks with your watermark appeared on a piracy platform then a judge might not believe your "they must have dumped these from my improperly secured computer" ...

> Then perhaps you should not publicly present your opinion/belief that it is perfectly legal to share drm-free ebooks with family members or close friends as if it were fact (as quoted above, and with no stipulations of jurisdictional variances at the time).

I agree with that, and I just edited my answer in #2 to make that more clear.
